backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/EakApplicationMutationService.kt

```diff
@@ -91,6 +105,26 @@ class EakApplicationMutationService {
         """.trimIndent()
     }
 
+    private fun generateApplicationApplicantMailMessage(
+        administrationName: String,
+        administrationBaseUrl: String,
+        personalData: PersonalData,
+        accessKey: String
+    ): String {
+        return """
+        Guten Tag ${personalData.forenames.shortText} ${personalData.surname.shortText},
+
+        Ihr Antrag zur Bayrischen Ehrenamtskarte wurde erfolgreich abgeschickt. 
+        
+        Sie können den Status Ihres Antrags unter folgendem Link einsehen und falls gewünscht ihren Antrag zurückziehen:
```

Comment:
[Optional]. If you reject this change, please make `Ihren` uppercase (second part of the sentence)
```suggestion
        Sie können den Status Ihres Antrags unter folgendem Link einsehen. Falls gewünscht, können Sie Ihren Antrag dort auch zurückziehen:
```